.travis.yml

```diff
@@ -9,9 +9,8 @@ install:
     - npm install
 script:
     - npm test
-services:
-  - mongodb
-cache: 
-  - npm install
 notifications:
   webhooks: $COVERALLS_URL
+before_script:
+  - export DISPLAY=:99.0
```

Comment:
What is the purpose of this?

Reply:
Karma launches a browser so this is creating a virtual browser